src/components/Home/styled.js

```diff
@@ -10,31 +10,20 @@ export const AppHeader = styled.header`
   display: flex;
   flex-direction: column;
   align-items: center;
-  justify-content: center;
   font-size: calc(10px + 2vmin);
-  color: white;
-`;
-
-export const AppLogo = styled.img`
```

Comment:
This is deleted, but is still imported in `Home/index.js`.

Reply:
Removed the import in Home/index.js